core/camel-core/src/test/java/org/apache/camel/component/file/stress/FileAsyncStressManualTest.java

```diff
@@ -33,6 +33,10 @@
 @DisabledOnOs(OS.WINDOWS)
 public class FileAsyncStressManualTest extends ContextTestSupport {
```

Comment:
Same note about manual tests I wrote for `FileAsyncStressReadLockRenameManualManualTest`